lib/ansible/modules/cloud/scaleway/scaleway_sshkey.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/python
+#
+# Scaleway SSH keys management module
+#
+# Copyright (C) 2018 Online SAS.
+# https://www.scaleway.com
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: scaleway_sshkey
+short_description: Scaleway SSH keys management module
+version_added: "2.6"
+author: Remy Leone (@sieben)
+description:
+    - This module manages SSH keys on Scaleway account
+      U(https://developer.scaleway.com)
+
+options:
+  state:
+    description:
+     - Indicate desired state of the SSH key.
+    required: true
+    choices:
+      - present
+      - absent
+  ssh_pub_key:
+    description:
+     - The public SSH key as a string to add.
+    required: true
+  oauth_token:
+    description:
+     - Scaleway OAuth token.
+    required: true
+  timeout:
+    description:
+    - Timeout for API calls
+    default: 30
+'''
+
+EXAMPLES = '''
+- name: "Add SSH key"
+  scaleway_sshkey:
+    ssh_pub_key: "ssh-rsa AAAA..."
+    state: "Present"
+
+- name: "Delete SSH key"
+  scaleway_sshkey:
+    ssh_pub_key: "ssh-rsa AAAA..."
+    state: "absent"
```

Comment:
maybe add an example showing an oauth_token in use?

Reply:
I've added an example with an explicit token